api/src/test/java/io/strimzi/test/StrimziRunner.java

```diff
@@ -585,81 +596,134 @@ private Statement withClusterOperator(Annotatable element,
-                    ObjectNode requests = new ObjectNode(factory);
-                    requests.put("cpu", "200m").put("memory", "512Mi");
-                    ObjectNode limits = new ObjectNode(factory);
-                    limits.put("cpu", "1000m").put("memory", "512Mi");
-                    resources.set("requests", requests);
-                    resources.set("limits", limits);
-                    containerNode.replace("resources", resources);
-                    containerNode.remove("resources");
-                    JsonNode ccImageNode = containerNode.get("image");
-                    ((ObjectNode) containerNode).put("image", TestUtils.changeOrgAndTag(ccImageNode.asText(), dockerOrg, dockerTag));
-                    for (JsonNode envVar : containerNode.get("env")) {
-                        String varName = envVar.get("name").textValue();
-                        // Replace all the default images with ones from the $DOCKER_ORG org and with the $DOCKER_TAG tag
-                        if (varName.matches("STRIMZI_DEFAULT_.*_IMAGE")) {
-                            String value = envVar.get("value").textValue();
-                            ((ObjectNode) envVar).put("value", TestUtils.changeOrgAndTag(value, dockerOrg, dockerTag));
-                        }
-                        // Set log level
-                        if (varName.equals("STRIMZI_LOG_LEVEL")) {
-                            String logLevel = System.getenv().getOrDefault("TEST_STRIMZI_LOG_LEVEL", "INFO");
-                            ((ObjectNode) envVar).put("value", logLevel);
-                        }
-                        // Updates default values of env variables
-                        for (EnvVariables envVariable : cc.envVariables()) {
-                            if (varName.equals(envVariable.key())) {
-                                ((ObjectNode) envVar).put("value", envVariable.value());
-                            }
+            boolean useHelmChart = cc.useHelmChart() || Boolean.parseBoolean(System.getProperty("useHelmChart", Boolean.FALSE.toString()));
+            if (useHelmChart) {
+                last = installOperatorFromHelmChart(element, last, cc);
+            } else {
+                last = installOperatorFromExamples(element, last, cc);
+            }
+        }
+        return last;
+    }
+
+    private Statement installOperatorFromHelmChart(Annotatable element, Statement last, ClusterOperator cc) {
+        String dockerOrg = System.getenv().getOrDefault("DOCKER_ORG", "strimzi");
+        String dockerTag = System.getenv().getOrDefault("DOCKER_TAG", "latest");
+
+        Map<String, String> values = Collections.unmodifiableMap(Stream.of(
+                entry("imageRepositoryOverride", dockerOrg),
+                entry("imageTagOverride", dockerTag),
+                entry("clusterOperator.image.pullPolicy", "Always"),
+                entry("clusterOperator.resources.requests.memory", "512Mi"),
```

Comment:
"512Mi" etc should probably be constants, so they get modified in tandem between the helm and examples installations

Reply:
I'll make them constants.